core/templates/pages/exploration-editor-page/services/exploration-data.service.spec.ts

```diff
@@ -104,8 +132,11 @@ describe('Exploration data service', function() {
       return deferred.promise;
     });
   }));
+  afterEach(() => {
+    httpTestingController.verify();
+  });
 
-  it('should autosave draft changes when draft ids match', function() {
+  it('should autosave draft changes when draft ids match', fakeAsync(() => {
     var errorCallback = jasmine.createSpy('error');
```

Comment:
Ditto regarding `let`.

Reply:
Will be done when we migrate this particular spec file.